internal/xds/translator/translator.go

```diff
@@ -509,10 +511,25 @@ func processXdsCluster(tCtx *types.ResourceVersionTable, httpRoute *ir.HTTPRoute
 		endpointType = EndpointTypeStatic
 	}
 
+	tsocket := &corev3.TransportSocket{}
```

Comment:
this logic should live inside `addXdsCluster`, with the suggestion of moving the upstream tls context into `DestinationSetting`, should be easier to build this inside that method

Reply:
moved